Cargo.toml

```diff
@@ -10,22 +10,23 @@ version = "0.3.0"
 [features]
 default = []
 fluent = ["fluent-templates"]
+env_logger = []
 
 [dependencies]
 clap = { version = "4", features = ["derive", "env", "unicode", "cargo"] }
 color-eyre = "0.6"
-env_logger = "0.10"
-fluent-templates = { version = "0.8", optional = true, default-features = false, features = ["tera"]}
+env_logger = "0.11"
+fluent-templates = { version = "0.9", optional = true, default-features = false, features = ["tera", "use-ignore"] }
```

Comment:
What is `use-ignore` for ?

Reply:
There are two features, `use-ignore` and `walkdir`, one of which *must* be enabled. The default feature list includes `use-ignore`, so this is not a change except that by dropping default features this dependency now doesn't need the `macros` feature to be compiled. I assume that was the point of this change...which makes sense since they are not being used in this context.